README.md

```diff
@@ -79,55 +87,28 @@ If you want to override Polly configuration, you can add configuration to `globa
 }
 ```
 
-*See all of the valid Polly options [in the Polly documentation](https://netflix.github.io/pollyjs/#/configuration).*
+_See all of the valid Polly options [in the Polly documentation](https://netflix.github.io/pollyjs/#/configuration)._
 
-You may also want to get at the global Polly instance. You can grab it from the `global` object in Node:
+You may also want to get at the global Polly instance. You can import it:
 
 ```js
-global.pollyContext.polly.server.get('/series')
+// if you need access to the pollyContext
+import { pollyContext } from '@spotify/polly-jest-presets';
+
+pollyContext.polly.server
+  .get('/series')
   .intercept((req, res) => res.sendStatus(200));
 ```
 
-*See all of the Polly API methods [in the Polly documentation](https://netflix.github.io/pollyjs/#/api).*
+_See all of the Polly API methods [in the Polly documentation](https://netflix.github.io/pollyjs/#/api)._
 
 ## Opinions
 
 This preset has a few opinions baked in. All of these are overridable by setting the `globals.pollyConfig` in your Jest config.
 
 ### Expire recordings often
 
-We think it's safer to expire recordings frequently. The default `expiresIn` is set to `"14d"` in this preset, and [we plan to update Polly](https://github.com/Netflix/pollyjs/issues/226) to support throwing errors when encountering expired recordings in `replay` mode.
-
-#### Enabling warning for recordings expiring soon
```

Comment:
Removing this intentionally?

Reply:
i did! since the default expiryStrategy is no to warn, we didn't need a precommit hook to warn. we can add back if you still think it's useful.